deployment-docs/clouds.md

```diff
@@ -0,0 +1,12 @@
+# Clouds
+
+Taskcluster is a multi-cloud system, and as such expects credentials for several cloud providers.
```

Comment:
Do we consider rabbitmq one of the cloud providers? If I was looking at these docs first when evaluating taskcluster, I might mistake the following for a complete list of external services you have to either make or rent in order to get a working TC...

Reply:
I don't think so -- and there's a separate section on Pulse, anyway.

Reply:
(In my mind, "cloud" means a provider of generalized cloud services like compute and storage -- Azure, GCP, AWS, Packet)